src/java/boa/functions/BoaAstIntrinsics.java

```diff
@@ -209,9 +210,12 @@ private static void openMap() {
 		final Configuration conf = new Configuration();
 		try {
 			final FileSystem fs = FileSystem.get(conf);
-			final Path p = new Path("hdfs://boa-njt/",
```

Comment:
I haven't coded it yet.  The plan was very similar to this though: create a configurable option, which defaults to 'boa-njt' and allow the user to override the HDFS root path via commandline.  Then all of these hard coded paths would simply use that configuration value.

Besides, the current code here is wrong because all 3 paths (ast, comments, issues) are using a single configuration value.  The sub-dirs are lost.